lib/nexmo_developer/db/schema.rb

```diff
@@ -2,15 +2,15 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# This file is the source Rails uses to define your schema when running `rails
-# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# This file is the source Rails uses to define your schema when running `bin/rails
```

Comment:
this migration is not really necessary, we already have users and I think a few tables from the server side tracking  were added by mistake